lms/djangoapps/ccx/api/v0/views.py

```diff
@@ -617,6 +703,17 @@ def patch(self, request, ccx_course_id=None):
                 if ccx_course_object.coach.id != coach.id:
                     old_coach = ccx_course_object.coach
                     ccx_course_object.coach = coach
+            if 'course_modules' in valid_input:
+                if valid_input.get('course_modules'):
+                    if not valid_course_modules(valid_input['course_modules'], master_course_key):
+                        return Response(
+                            status=status.HTTP_400_BAD_REQUEST,
+                            data={
+                                'error_code': 'course_module_list_not_belonging_to_master_course'
+                            }
+                        )
+                # course_modules to be stored in a json stringified field
+                ccx_course_object.structure_json = json.dumps(valid_input.get('course_modules'))
```

Comment:
yes, the idea is that you can remove the field with the patch passing `None` as value for `course_modules `